stl/inc/xloctime

```diff
@@ -93,37 +93,38 @@ public:
         _State = ios_base::goodbit;
 
         for (; _Fmtfirst != _Fmtlast; ++_Fmtfirst) {
-            if (_Ctype_fac.narrow(*_Fmtfirst) != '%') { // match literal element
+            if (_State != ios_base::goodbit) {
+                // [locale.time.get.members] p8.2

```

Comment:
We try to cite the Working Paper in addition to the stable name and paragraph, because paragraphs are commonly renumbered (and wording sometimes moves between stable names).

(Applies to the other citations.)